manager/controllers/app/m4dapplication_controller.go

```diff
@@ -102,25 +110,25 @@ func (r *M4DApplicationReconciler) Reconcile(req ctrl.Request) (ctrl.Result, err
 	return ctrl.Result{}, nil
 }
 
-func checkBlueprintStatus(applicationContext *app.M4DApplication, blueprint *app.Blueprint) {
+func checkBlueprintStatus(applicationContext *app.M4DApplication, status ResourceStatus) {
 	applicationContext.Status.DataAccessInstructions = ""
 	applicationContext.Status.Ready = false
 	if hasError(applicationContext) {
 		return
 	}
-	if blueprint.Status.Error != "" {
-		setCondition(applicationContext, "", blueprint.Status.Error, "Blueprint", true)
+	if status.Error != "" {
+		setCondition(applicationContext, "", status.Error, "Orchestration", true)
 		return
 	}
-	if blueprint.Status.Ready {
+	if status.Ready {
 		applicationContext.Status.Ready = true
-		applicationContext.Status.DataAccessInstructions = blueprint.Status.DataAccessInstructions
```

Comment:
As we discussed, future PR should have a map of user info instead of this, since it is specific to a read flow using arrow flight.

Reply:
OK. Will be changed in the future.